finance_manager/src/main/java/com/manager/finance/mapstruct/mapper/WalletMapper.java

```diff
@@ -6,17 +6,17 @@
 import java.util.Formatter;
 import java.util.List;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
 import org.mapstruct.MappingTarget;
 import org.mapstruct.Named;
 
 @Mapper(componentModel = "spring")
 public interface WalletMapper {
 
-
-
     @Named("intToStringDouble")
-    public static String intToStringDouble(int amount) {
-        return new Formatter().format("%.2f", ((double) (amount)) / 100).toString();
+    static String intToStringDouble(Integer amount) {
+        return amount == null ? "0.00"
+                : new Formatter().format("%.2f", ((double) (amount)) / 100).toString();
```

Comment:
The same, we can move this method in Utility class and reuse it

Reply:
created